installer/install.sh

```diff
@@ -69,15 +69,12 @@ argocd app sync science-platform \
   --port-forward-namespace argocd
 
 echo "Syncing critical early applications"
-argocd app sync nginx-ingress \
-  --port-forward \
-  --port-forward-namespace argocd || true
```

Comment:
I see that you removed the `nginx-ingress` from the critical early applications here.  That's getting the installer  stuck when syncing the science platform apps. If I cancel the installer, sync the the `nginx-ingress` app manually and run the installer again it goes on and finishes the installation.

Reply:
Ah fantastic, okay, I was wondering that.  I thought it was tied to the cert manager stuff, and since using DNS it shouldn't need that anymore.  I just ran on GKE the rogue-two setup and that doesn't hang - but that doesn't include chronograph and some of the other components too.  Maybe those require some ingress to do something.  I'm going to try to deploy that one (but not mess with the DNS) and see if I can repro, and then add this back and see if it works.

Thanks so much!

Reply:
Okay added this back in.